ipatests/prci_definitions/temp_commit.yaml

```diff
@@ -61,14 +61,14 @@ jobs:
         timeout: 1800
         topology: *build
 
-  fedora-latest/temp_commit:
-    requires: [fedora-latest/build]
+  fedora-rawhide/TestRSNPKIConfig:
+    requires: [fedora-rawhide/build]
```

Comment:
If the job requires `fedora-rawhide/build`, then the name of the build job on line 50 should also be `fedora-rawhide/build`, not `freeipa/ci-master-frawhide`, and the build job needs to use `ci-master-frawhide`. You can paste the job definition from `ipatests/prci_definitions/nightly_rawhide.yaml` to avoid substitution errors:
```
  fedora-rawhide/build:
    requires: []
    priority: 100
    job:
      class: Build
      args:
        git_repo: '{git_repo}'
        git_refspec: '{git_refspec}'
        template: &ci-master-frawhide
          name: freeipa/ci-master-frawhide
          version: 0.8.0
        timeout: 1800
        topology: *build

  fedora-rawhide/testRSNPKIConfig:
    requires: [fedora-rawhide/build]
    priority: 50
    job:
      class: RunPytest
      args:
        build_url: '{fedora-rawhide/build_url}'
        update_packages: True
        test_suite: test_integration/test_random_serial_numbers.py::TestRSNPKIConfig
        template: *ci-master-frawhide
        timeout: 10800
        topology: *master_3repl_1client

```